astropy/cosmology/core.py

```diff
@@ -1677,7 +1677,7 @@ def efunc(self, z):
         return np.sqrt(zp1 ** 2 * ((Or * zp1 + Om0) * zp1 + Ok0) + Ode0)
 
     def inv_efunc(self, z):
-        r""" Function used to calculate :math:`\frac{1}{H_z}`.
+        r""" Function used to calculate :math:`\\frac{1}{H_z}`.
```

Comment:
This isn't necessary here since it's a "raw" string (the `r` before the opening quotes).  It seems to be rendering fine too: http://docs.astropy.org/en/latest/api/astropy.cosmology.LambdaCDM.html#astropy.cosmology.LambdaCDM.inv_efunc